### ISSUE TITLE
rules_nodejs is broken on Windows, Bazel CI didn't catch it

### ISSUE THREAD

**Issue by meteorcloudy** [bazelbuild/rules_nodejs#582]
# 🐞 bug report

### Is this a regression?

Yes, rules_nodejs no longer builds on Windows

### Description

Rules_nodejs is red in downstream test on Bazel CI:
https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/826#40bcf682-9310-4cb7-8215-0b09f9f601bf

First I thought it was a Bazel bug, but while bisecting, even a previous good bazel commit fails with the same error.
https://buildkite.com/bazel/culprit-finder/builds/117#9cd512ab-6511-4941-8140-779f1bc47655

Then I tried to reproduce on my local Windows machine with Bazel 0.22.0, I got the same error.
After a local bisecting, I found https://github.com/bazelbuild/rules_nodejs/pull/575 as the culprit.

I suspect we didn't catch this on [presubmit](https://buildkite.com/bazel/rules-nodejs-nodejs/builds/1513#148f49c5-f042-40cb-8bdf-d70da181839c) because the actions were remote cached.
This means we have **a hermetic problem** in either rules_nodejs rule or Bazel, please help investigate.

## 🔬 Minimal Reproduction

With Bazel 0.22.0
```
bazel run --verbose_failures --announce_rc @nodejs//:yarn
bazel build --verbose_failures --announce_rc //...
```

## 🔥 Exception or Error

<pre><code>

ERROR: D:/b/bk-windows-java8-5s95/bazel-downstream-projects/rules_nodejs/internal/e2e/rollup/BUILD.bazel:4:1: Couldn't build file internal/e2e/rollup/bundle.js: SkylarkAction internal/e2e/rollup/bundle.js failed (Exit 2): tsc.exe failed: error executing command
--
  | cd D:/b/vnydp666/execroot/build_bazel_rules_nodejs
  | bazel-out/host/bin/external/build_bazel_rules_nodejs/internal/rollup/tsc.exe --target es5 --allowJS bazel-out/x64_windows-fastbuild/bin/internal/e2e/rollup/bundle.es6.js --outFile bazel-out/x64_windows-fastbuild/bin/internal/e2e/rollup/bundle.js
  | Execution platform: @bazel_tools//platforms:host_platform
  | ../../../bk-windows-java8-5s95/bazel-downstream-projects/rules_nodejs/node_modules/@types/node/util.d.ts(117,88): error TS2304: Cannot find name 'Symbol'.

</code></pre>

## 🌍  Your Environment

**Operating System:**

<pre>
  <code>
Windows
  </code>
</pre>

**Output of `bazel version`:**

<pre>
  <code>
0.22.0
  </code>
</pre>

**Rules version (SHA):**

<pre>
  <code>
8f1bcccc7e6b07f1b3ae55f39d9e94b4aedd12b5
  </code>
</pre>

**Comment by meteorcloudy** [comment]
@alexeagle This also means rules_nodejs 0.26.0 release doesn't work on Windows.

**Comment by meteorcloudy** [comment]
I can reproduce the issue on CI after disabling remote cache in tests
https://buildkite.com/bazel/rules-nodejs-nodejs/builds/1535#8a7323c9-7102-4f1e-98b1-e865bbc4fedb

**Comment by alexeagle** [comment]
Maybe we should disable remote caching until we have a way to test the hermeticity of rules?

**Comment by meteorcloudy** [comment]
Yes, I already did that for running the tests, there is no easy way to disable remote cache for the build currently. (Maybe we should add one @fweikert)

@alexeagle @gregmagolan Please work on a fix for this, otherwise the presubmit of rules_nodejs will always be red.

**Comment by gregmagolan** [comment]
@meteorcloudy Thanks for reporting. I'll TAL today or tomorrow.

**Comment by gregmagolan** [comment]
I tried reproducing on my local Windows machine with no luck

**Comment by gregmagolan** [comment]
That's strange. I checked the latest builds https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/840 & https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/839 and rules_nodejs is all green. Don't know what caused the failure or how you managed to reproduce it locally @meteorcloudy. Closing this as its now passing at head but feel free to re-open if it shows up again.
